src/apis/Meeting/index.ts

```diff
@@ -2,45 +2,55 @@ import {
   APICreateMeetingReqeust,
   APIEntireMeetingList,
   APIMeetingGroup,
-} from "@/types/meeting";
-import { APIMeetingDetailCommentsList } from "@/types/meetingDetailCommentsList";
-import { APIMeetingDetail } from "@/types/meetingDetail";
-import { publicApi } from "../core/axios";
+} from '@/types/meeting';
+import { APIMeetingDetailCommentsList } from '@/types/meetingDetailCommentsList';
+import { APIMeetingDetail } from '@/types/meetingDetail';
+import { publicApi } from '../core/axios';
 
 const MeetingAPI = {
   getEntireMeetingList: () =>
     publicApi.get<APIEntireMeetingList>(`/api/book-groups`, {
       params: {
         pageSize: 10,
         groupCursorId: 999,
-        sortDirection: "DESC",
+        sortDirection: 'DESC',
       },
     }),
 
   createMeeting: ({ meeting }: { meeting: APICreateMeetingReqeust }) =>
-    publicApi.post("/api/book-groups", meeting),
+    publicApi.post('/api/book-groups', meeting),
 
   getMeetingDetailInfo: ({
     bookGroupId,
   }: {
-    bookGroupId: APIMeetingGroup["bookGroupId"];
+    bookGroupId: APIMeetingGroup['bookGroupId'];
   }) => publicApi.get<APIMeetingDetail>(`/api/book-groups/${bookGroupId}`),
 
   getMeetingDetailCommentsList: ({
     bookGroupId,
   }: {
-    bookGroupId: APIMeetingGroup["bookGroupId"];
+    bookGroupId: APIMeetingGroup['bookGroupId'];
   }) =>
     publicApi.get<APIMeetingDetailCommentsList>(
       `/api/book-groups/${bookGroupId}/comments`,
       {
         params: {
           groupCommentCursorId: 100,
           pageSize: 10,
-          sortDirection: "DESC",
+          sortDirection: 'DESC',
         },
       }
     ),
+  createMeetingComment: ({
+    bookGroupId,
+    comment,
+  }: {
+    bookGroupId: APIMeetingGroup['bookGroupId'];
+    comment: string;
+  }) =>
+    publicApi.post(`/api/book-groups/${bookGroupId}/comment`, {
+      comment: `${comment}`,
```

Comment:
p2; 
```suggestion
      comment,
```

이렇게 수정할 수 있을 것 같아요